include/openenclave/bits/sgx/sgxproperties.h

```diff
@@ -330,20 +330,21 @@ typedef struct _oe_sgx_enclave_properties
  * application, to create a 0-base enclave.
  */
 /*
- * Uncomment the lines below if you wish to create enclave
- * at zero base address. The enclave is created with
- * base address = start address by default.
- * NOTE: SGX_ENCLAVE_ZERO_BASE has to be defined before
- * including headers.
+ * To create enclave at zero base address, uncomment the lines.
```

Comment:
A developer should not need to edit files under the include directory in order to enable features, that's what parameters passed to `oe_create_enclave()` are for.   We should not have commented out code.

Reply:
Hi @dthaler, these #defines will have to be added as a part of the application's enc/enc.cpp. I have not written a test case yet, and have it here as a reference/ reminder. These comments will be deleted once the code is ready for review.